its/src/org/sonarlint/eclipse/its/StandaloneAnalysisTest.java

```diff
@@ -137,17 +137,25 @@ public void analyze_automatic_workspace_build_disabled() {
   /** See SLE-854: JDT tries to find files with 'Java-like' extensions even if they're not Java */
   @Test
   public void test_jdt_java_like_extension_COBOL() {
-    new JavaPerspective().open();
-    var rootProject = importExistingProjectIntoWorkspace("connected", "connected");
-
-    var cobolFile = rootProject.getResource("Test.cbl");
-    openFileAndWaitForAnalysisCompletion(cobolFile);
-
-    // even when no language found, the Secrets analyzer should at least give it a shot^^
-    var consoleText = new SonarLintConsole().getConsoleView().getConsoleText();
-    assertThat(consoleText)
-      .contains("Execute Sensor: TextAndSecretsSensor")
-      .doesNotContain("File 'Test.cbl' excluded by 'JavaProjectConfiguratorExtension'");
+    // INFO: Here we check for the IDE-specific logs, so we have to enable them!
+    var consoleView = new SonarLintConsole();
+    try {
+      consoleView.enableIdeSpecificLogs(true);
+
+      new JavaPerspective().open();
+      var rootProject = importExistingProjectIntoWorkspace("connected", "connected");
+
+      var cobolFile = rootProject.getResource("Test.cbl");
+      openFileAndWaitForAnalysisCompletion(cobolFile);
+
+      // even when no language found, the Secrets analyzer should at least give it a shot^^
+      var consoleText = new SonarLintConsole().getConsoleView().getConsoleText();
+      assertThat(consoleText)
+        .contains("Execute Sensor: TextAndSecretsSensor")
+        .doesNotContain("File 'Test.cbl' excluded by 'JavaProjectConfiguratorExtension'");
```

Comment:
no big deal but this test does not really guarantee that the IDE specific logs will work, it checks that the log line will NOT be there - either it's really not there or the logging is not working :D 

If possible, maybe we could also assert that something from the IDE logs IS there

Reply:
Good idea, I see what a reliable log message we could use

Reply:
Done